packages/dart/cryptography/lib/src/cryptography/base_cryptography_service.dart

```diff
@@ -3,7 +3,7 @@
 import 'dart:convert';
 import 'dart:math';
 import 'dart:typed_data';
-import 'package:affinidi_tdk_cryptography/src/models/verify_jwt_result.dart';
+import 'package:affinidi_tdk_cryptography_dart/src/models/verify_jwt_result.dart';
```

Comment:
Can we do a relative import here? It is bad practice for the package to import itself. https://dart.dev/tools/pub/create-packages#importing-library-files